### PR TITLE
gobang: Add version 0.1.0-alpha.5

### PR DESCRIPTION
**Pull request by TaKO8Ki**
Add gobang

[gobang](https://github.com/TaKO8Ki/gobang): A cross-platform TUI database management tool written in Rust

### CODE REVIEW

**Comment by TaKO8Ki** [comment]
scoop tries to download v0.1.0 in CI, but there isn't.

File: bucket/gobang.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    "homepage": "https://github.com/TaKO8Ki/gobang",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/TaKO8Ki/gobang/releases/download/v0.1.0-alpha.5/gobang-0.1.0-alpha.5-x86_64-pc-windows-msvc.zip",
+            "hash": "4fb7d76776cc778e047b25eb460cdf76745642b937efa07737d0ee077b4541a5"
+        }
+    },
+    "bin": "gobang.exe",
+    "checkver": "github",
```
Comment (issaclin32) [inline bucket/gobang.json:13]:
```suggestion
    "checkver": {
        "github": "https://github.com/TaKO8Ki/gobang",
        "regex": "download/v([\\w.-]+)"
    },
```

Comment (issaclin32) [reply bucket/gobang.json:13]: *checkver* only matches `([\\d.]+)` by default. Therefore `"checkver": "github"` does not work.

Comment (TaKO8Ki) [reply bucket/gobang.json]: Thank you for your advice.

File: bucket/gobang.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    },
+    "bin": "gobang.exe",
+    "checkver": {
+        "github": "https://github.com/TaKO8Ki/gobang",
+        "regex": "download/v([\\w.-]+)"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/TaKO8Ki/gobang/releases/download/v$version/gobang-$version-x86_64-pc-windows-msvc.zip"
```
Comment (issaclin32) [inline bucket/gobang.json:20]:
```suggestion
                "url": "https://github.com/TaKO8Ki/gobang/releases/download/v$version/gobang-$version-x86_64-pc-windows-msvc.zip",
                "extract_dir": "gobang-$version-x86_64-pc-windows-msvc"
```

File: bucket/gobang.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
+{
+    "version": "0.1.0-alpha.5",
+    "description": "A cross-platform TUI database management tool written in Rust",
+    "homepage": "https://github.com/TaKO8Ki/gobang",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/TaKO8Ki/gobang/releases/download/v0.1.0-alpha.5/gobang-0.1.0-alpha.5-x86_64-pc-windows-msvc.zip",
+            "hash": "4fb7d76776cc778e047b25eb460cdf76745642b937efa07737d0ee077b4541a5"
```
Comment (issaclin32) [inline bucket/gobang.json:9]:
```suggestion
            "hash": "4fb7d76776cc778e047b25eb460cdf76745642b937efa07737d0ee077b4541a5",
            "extract_dir": "gobang-0.1.0-alpha.5-x86_64-pc-windows-msvc"
```

**Review by issaclin32 — APPROVED** [review]

**Pull request merged**